package.json

```diff
@@ -8,10 +8,7 @@
   "packageManager": "yarn@1.22.19",
   "devDependencies": {
     "@eslint/js": "^9.2.0",
-    "@typescript-eslint/eslint-plugin": "^7.8.0",
-    "@typescript-eslint/parser": "^7.8.0",
     "eslint": "^9.2.0",
-    "eslint-config-prettier": "^8.8.0",
```

Comment:
Actually, I think we should leave this and update it to the version that works with flat config so it turns off all of the styling rules that are covered by Prettier.

Reply:
Do you think some stylistic rule conflicts with prettier, because `ts-eslint` state on their website: "Neither typescript-eslint nor ESLint core enable any formatting-related rules in any recommended presets."

So that's why I thought its safe to just omit `eslint-config-prettier` completely.